internal/core/services/scheduler_manager/scheduler_manager.go

```diff
@@ -73,6 +75,23 @@ func (s *SchedulerManager) GetAllSchedulers(ctx context.Context) ([]*entities.Sc
 	return s.schedulerStorage.GetAllSchedulers(ctx)
 }
 
+func (s *SchedulerManager) AddRooms(ctx context.Context, schedulerName string, amount int32) (*operation.Operation, error) {
```

Comment:
@luizmiranda7 would it make sense to have a unit test for this new func? with the initial struct of AddRoomsDefinition that you have created we would be able to write a test for this func using mocks, right?